dali/operators/image/remap/remap.cuh

```diff
@@ -62,18 +61,16 @@ template<typename StorageBackend, typename T, int ndims>
 T **GetGpuAccessibleTensors(TensorListView <StorageBackend, T, ndims> tlv,
                             dali::kernels::DynamicScratchpad &ds, cudaStream_t stream) {
   using DataType = T *;
-  auto data_size = tlv.num_samples() * sizeof(DataType);
-  DataType *ret = ds.template AllocatePinned<DataType>(data_size);
-  CUDA_CALL(cudaMemcpyAsync(ret, tlv.data.data(), data_size, cudaMemcpyDefault, stream));
+  DataType *tmp = ds.template ToPinned(tlv.data);
+  DataType *ret = ds.template ToGPU(stream, span<DataType>(tmp, tlv.num_samples()));
   return ret;
 }
 
 
 size_t *GetGpuAccessibleSampleSizes(size_t *sample_sizes, size_t n_samples,
                                     dali::kernels::DynamicScratchpad &ds, cudaStream_t stream) {
-  auto data_size = n_samples * sizeof(size_t);
-  auto ret = ds.template AllocatePinned<size_t>(data_size);
-  CUDA_CALL(cudaMemcpyAsync(ret, sample_sizes, data_size, cudaMemcpyDefault, stream));
+  auto tmp = ds.template ToPinned(span<size_t>(sample_sizes, n_samples));
+  auto ret = ds.template ToGPU(stream, span<size_t>(tmp, n_samples));
```

Comment:
I think that `ToContiguousGPU` would also work (it copies to pinned mem first)

Reply:
Done